File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
		//品牌服务
		app.service("brandService",function($http){
			//查询品牌列表
			this.findAll=function(){
				return $http.get('../brand/findAll.do');
			}
			//分页
			this.findPage=function(page,size){
				return $http.get('../brand/findPage.do?page='+page +'&size='+size);
			}
			//增加
			this.add=function(entity){
				return $http.post('../brand/add.do',entity);
			}
			//更新
			this.update=function(entity){
				return $http.post('../brand/update.do',entity);
			}
			//查询实体
			this.findOne=function(id){
				return $http.get('../brand/findOne.do?id='+id);
			}
			//删除
			this.dele=function(Ids){
				return $http.get('../brand/delete.do?ids='+Ids);
			}
			//搜索
			this.search=function(page,size,searchEntity){
				return $http.post('../brand/search.do?page='+page +'&size='+size, searchEntity);
			}
			//品牌列表
			this.selectOptionList=function(){
				return $http.get('../brand/selectOptionList.do');
			}
		});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;
/**
 * 品牌服务层接口
 * @author 57473
 *
 */

import entity.PageResult;
public interface BrandService {
	/**
	 * 查询品牌
	 * @return
	 */
	public List<TbBrand> findAll();
	/**
	 * 品牌分页
	 * @param pageNum
	 * @param pageSize
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	/**
	 * 增加
	 * @param tbBrand
	 */
	public void add(TbBrand tbBrand);
	/**
	 * 根据id查询实体类
	 * @param id
	 * @return
	 */
	public TbBrand findOne(Long id);
	/**
	 * 修改
	 * @param tbBrand
	 */
	public void update(TbBrand tbBrand);
	/**
	 * 删除
	 * @param ids
	 */
	public void delete(Long[] ids);
	/**
	 * 条件查询
	 * @param pageNum
	 * @param pageSize
	 * @param brand
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize,TbBrand brand);
	/**
	 * 下拉框数据
	 * @return
	 */
	public List<Map> selectOptionList();
	
}
